vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -0,0 +1,90 @@
+---
+  - name: Label BMO CRDs.
+    shell: 'kubectl label --overwrite crds baremetalhosts.metal3.io {{ item }} --overwrite '
+    with_items:
+       - clusterctl.cluster.x-k8s.io=""
+       - cluster.x-k8s.io/provider="metal3"
+    
+  - name: Obtain target cluster kubeconfig
+    shell: "kubectl get secrets {{ CLUSTER_NAME }}-kubeconfig -n {{ NAMESPACE }} -o json | jq -r '.data.value'| base64 -d > /tmp/target.yaml"
+    register: target_kubeconfig
+
+  - name: Create namespace
+    k8s:
+      name: "{{ NAMESPACE }}"
+      kind: Namespace
+      state: present
+    environment:
+      KUBECONFIG: "/tmp/target.yaml"
+
+  # Install BMO
+  - name: Install Baremetal Operator
+    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/deploy/ironic-outside-config/ | kubectl --kubeconfig=/tmp/target.yaml apply -n {{ NAMESPACE }} -f -"
+
+  - name: Pivoting test
+    delegate_to: "{{ CLUSTER_APIENDPOINT_IP }}"
+    vars:
+      ansible_user: "{{ IMAGE_USERNAME }}"
+      ansible_ssh_private_key_file: "{{ SSH_PRIVATE_KEY }}"
+      ansible_python_interpreter: /usr/bin/python3
+    become: yes
+    become_user: "{{ IMAGE_USERNAME }}"
+    block:
+      - name: Clone Baremetal Operator
+        git:
+          repo: 'https://github.com/metal3-io/baremetal-operator.git'
+          dest: "/home/{{ IMAGE_USERNAME }}/baremetal-operator"
+            
+        # Replacing the name of the bridge
+      - name: Replace the Provisioning Interface in ironic_ci.env
+        lineinfile:
+          path: /home/{{ IMAGE_USERNAME }}/baremetal-operator/deploy/ironic_ci.env
+          regexp: "^PROVISIONING_INTERFACE="
+          line: "PROVISIONING_INTERFACE={{ IRONIC_ENDPOINT_BRIDGE }}"
+        
+        # Install ironic
+      - name: Install ironic
+        shell: "bash /home/{{ IMAGE_USERNAME }}/baremetal-operator/tools/run_local_ironic.sh"
+        environment:
+          CONTAINER_RUNTIME: "{{ CONTAINER_RUNTIME }}"
+        become: yes
+        become_user: root
```

Comment:
why is this running out of the target cluster ? you could use the BMO-keepalived kustomization to not need to run it outside of the cluster

Reply:
This is running in target cluster. See line 25

Reply:
no, I mean why not running it in k8s ? why running it locally ?

Reply:
This is a test-case, I don't see any problem of running the ironic locally or am I missing something?

Reply:
there is no problem, it's just that you don't really need to do all, as you could very simply running it in cluster by using the other kustomization